.cloudbuild/library_generation/library_generation_airlock.Dockerfile

```diff
@@ -34,29 +34,8 @@ RUN mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip -Dfmt.skip
 RUN cp "/root/.m2/repository/com/google/api/gapic-generator-java/${DOCKER_GAPIC_GENERATOR_VERSION}/gapic-generator-java-${DOCKER_GAPIC_GENERATOR_VERSION}.jar" \
   "./gapic-generator-java.jar"
 
-# 3.20.1
-FROM us-docker.pkg.dev/artifact-foundry-prod/docker-3p-trusted/alpine@sha256:dabf91b69c191a1a0a1628fd6bdd029c0c4018041c7f052870bb13c5a222ae76 as glibc-compat
-
-RUN apk update && apk add git sudo
-# This SHA is the latest known-to-work version of this binary compatibility tool
-ARG GLIB_MUS_SHA=e94aca542e3ab08b42aa0b0d6e72478b935bb8e8
-WORKDIR /home
-
-# Install compatibility layer to run glibc-based programs (such as the
-# grpc plugin).
-# Alpine, by default, only supports musl-based binaries, and there is no public
-# downloadable distribution of the grpc plugin that is Alpine (musl) compatible.
-# This is one of the recommended approaches to ensure glibc-compatibility
-# as per https://wiki.alpinelinux.org/wiki/Running_glibc_programs
-RUN git clone https://gitlab.com/manoel-linux1/GlibMus-HQ.git
-WORKDIR /home/GlibMus-HQ
-# We lock the tool to the latest known-to-work version
-RUN git checkout "${GLIB_MUS_SHA}"
-RUN chmod a+x compile-x86_64-alpine-linux.sh
-RUN sh compile-x86_64-alpine-linux.sh
-
-# 3.12.7-alpine3.20
-FROM us-docker.pkg.dev/artifact-foundry-prod/docker-3p-trusted/python@sha256:b83d5ec7274bee17d2f4bd0bfbb082f156241e4513f0a37c70500e1763b1d90d as final
+# 3.12.3-slim-bookworm
```

Comment:
Is this older version of the public image above? docker.io/library/python:3.13.2-slim